mailpoet/mailpoet
# Non-static scheduler

Fixes [MAILPOET-4252]

[MAILPOET-4252]: https://mailpoet.atlassian.net/browse/MAILPOET-4252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Test steps:
1. Go to MailPoet > Emails
2. Schedule a new simple newsletter to be sent to your email today (remember the date/time)
3. Go to MailPoet Debug, copy the password shown and click Adminer link
4. Paste the password and click Login
5. Click [wp_mailpoet_scheduled_tasks](https://gs.qawp.net/wp-content/plugins/mailpoet-debug-trunk/adminer.php?server=localhost&username=admin_gsqawpuser&db=admin_gsqawpdb&table=wp_mailpoet_scheduled_tasks) table when inside
6. Click Select data link
7. Click twice time `scheduled_at` row to sort it to the last newsletter
8. You need to find your scheduled newsletter here, look for the date/time and seek for what you scheduled in step 2. When you find it, click Edit link to edit that `sending` newsletter
9. Now just edit the scheduled_at date to be sent in the next few minutes... (just change the time to be 5 mins in front of now)
10. Click Save button
11. Now just wait for it to be sent (at that time that you set) and check inbox to receive the scheduled newsletter

Note: Please remember these steps as it will be required in the future to edit your scheduled newsletters and execute them sooner.
